javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentArgUtil.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import javax.annotation.Nullable;
+
+public class AgentArgUtil {
+
+  private AgentArgUtil() {}
+
+  @SuppressWarnings("SystemOut")
+  public static void setSystemProperties(@Nullable String agentArgs) {
+    boolean debug = false;
+    if (agentArgs != null && !agentArgs.isEmpty()) {
+      String[] options = agentArgs.split(";");
+      for (String option : options) {
+        String[] keyValue = option.split("=");
+        if (keyValue.length == 2) {
+          if (keyValue[0].equals("otel.javaagent.debug")) {
+            debug = Boolean.parseBoolean(keyValue[1]);
+          } else {
+            System.setProperty(keyValue[0], keyValue[1]);
+            if (debug) {
+              System.out.println("Setting property [" + keyValue[0] + "] = " + keyValue[1]);
+            }
+          }
```

Comment:
do you want to pass along `otel.javaagent.debug` as well?
```suggestion
          System.setProperty(keyValue[0], keyValue[1]);
          if (debug) {
            System.out.println("Setting property [" + keyValue[0] + "] = " + keyValue[1]);
          }
```